### PR TITLE
Allows downloading files >1GB

### PR DESCRIPTION
**Pull request by mdimura**

### CODE REVIEW

File: conf/nginx-proxy.conf [hunk 1]
```diff
@@ -8,6 +8,9 @@ location YNH_LOCATION {
   proxy_set_header  X-Forwarded-Host $server_name;
   proxy_set_header  X-Forwarded-Port $server_port;
   
+  #allow downloading files >1GB
+  proxy_buffering off;
```
Comment (zamentur) [inline conf/nginx-proxy.conf]: WIth this options the include of the yunohost square won't work i think...

Comment (zamentur) [reply conf/nginx-proxy.conf]:
```suggestion
 client_max_body_size 10G;
```

Comment (mdimura) [reply conf/nginx-proxy.conf]: In that case, I suggest considering a larger limit, like 100G, larger than a Blu-ray. I added a new commit.

**Comment by zamentur** [comment]
The PR https://github.com/YunoHost-Apps/redirect_ynh/pull/37 allow the user to specify the limit needed (in order to avoid security issues related to put the value systematically too high).

**Pull request closed without merging**